fullscreen.bs

```diff
@@ -348,15 +359,10 @@ attribute's getter must run these steps:
    <var>doc</var>.
    <!-- cross-process -->
 
-   <li><p>If <var>resize</var> is true and <var>topLevelDoc</var> is not in <var>exitDocs</var>,
+   <li><p>If <var>resize</var> is true and <var>topLevelDoc</var> is either not in
+   <var>exitDocs</var>, or not a <a>simple fullscreen document</a>,
    <a>fully exit fullscreen</a> <var>topLevelDoc</var>, reject <var>promise</var> with a
```

Comment:
I'm curious if Gecko actually has this "fully exit fullscreen" step when exiting? In Blink, at least after the (now reverted) attempt to align with the spec, this didn't make sense. This is the topic of https://github.com/whatwg/fullscreen/pull/65

I'm fine with this change if it's just incidental of course, and not the central point of this PR.